src/IO/H5/Python/ExtendConnectivityData.py

```diff
@@ -0,0 +1,62 @@
+# Distributed under the MIT License.
```

Comment:
Please add only the bindings in this PR, not the CLI

Reply:
Is there any simple way that I can delete this file from the PR?

Reply:
You'll have to do an interactive rebase and then "edit" the first commit. We can do a hangout if that's helpful

Reply:
I think a hangout would be helpful. Thank you!